packages/plugins/other/jsdoc-operations/src/index.ts

```diff
@@ -0,0 +1,118 @@
+import { PluginFunction } from '@graphql-codegen/plugin-helpers';
+import { printSchema, parse, visit } from 'graphql';
+
+type NamedType = {
```

Comment:
I think maybe you can use the visitor to return directly the output string, it might simplify the implementation because you'll be able to skip the need to those proxy types.

Reply:
Was actually thinking about reusing `createTypedef` at some later point for other plugins as well, since the creation of a JSDoc typedef is consistent across those uses. I'd prefer a bit more extensive interfacing over having to duplicate the printing behavior :). What do you think?